modules/options-for-notebook-server-environments.adoc

```diff
@@ -30,8 +30,14 @@ endif::[]
 +
 After you start a notebook image, you can check which Python packages are installed on your notebook server and which version of the package you have by running the `pip` tool in a notebook cell.
 +
-The following table shows the package versions used in the available notebook images:
-+
+The following table shows the package versions used in the available notebook images.
+ifndef::upstream[]
+[IMPORTANT]
+====
+Notebook images denoted with `(Technology Preview)` in this table are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see Technology Preview Features Support Scope.
```

Comment:
minor comment - I did not catch this in my initial draft - second sentence - change "them" to "Technology Preview features":
Red Hat does not recommend using Technology Preview features in production.

Reply:
Good shout, have updated.